### ISSUE TITLE
Bring back the "Hide Chat Header" feature

### ISSUE THREAD

**Issue by DmitryScaletta** [night/BetterTTV#3688]
I used it to get more space for chat messages.

Twitch removed rooms but this feature was still useful for me and maybe for some other users.

For example my display resolution is 1440p and I zoom up twitch to 200% and hiding the chat header allows me to see more messages because it takes almost 10% of the chat.

_Commit:_ https://github.com/night/BetterTTV/commit/0b8d7bf89514ef6e1564032031991c873a6719d2

**Comment by Nareese** [comment]
Only night can say for sure, but I expect it's fairly unlikely this option will make a return.  In addition to removing important functionality (the user list button), removing the container for the header also breaks the layout of the right side "collapse" button in a way that is non-trivial to fix (see #3679 for more details on that).

In the interim, or in the event that it is not returned, you can either do what the user in that issue did, and use your ad-blocker to hide the header and the collapse button, or you can install a third party styling extension (Stylus recommended), create a style override for twitch.tv, and use the following rule to achieve the same effect as the old option + hiding the collapse button:

```
.room-selector .rooms-header, .right-column__toggle-visibility {
    display: none !important;
}
```

Please be aware that Twitch is likely to change the classes on these elements from time to time, so this "fix" will probably eventually break, and you'll have to figure out the new classes to maintain it.

**Comment by night** [comment]
unfortunately the feature should no longer exist because Twitch baked required buttons into the header. i'd rather remove it than deal with users complaining buttons are missing (which is what was happening)
